CHANGELOG.md

```diff
@@ -0,0 +1,188 @@
+## Changelog
+
+### 2.21 (Mar 7, 2019)
```

Comment:
nit: It would be great if you would move all of the release dates onto the next line in ISO 8601 date format, like this:

```suggestion
### 2.21

 Release date: 2019-03-07
```